e2e/src/context.ts

```diff
@@ -26,6 +27,7 @@ export interface Context {
     data?: object;
   };
   sessionIds?: string[];
+  paymentRequest?: Partial<PaymentRequest>;
```

Comment:
nit: This isn't wrong but it doesn't communicate what the interface expects. My preference is for types as documentation that state the expected structure explicitly.

Reply:
No this is totally fair - it's actually ended up as not used, so this has been tidied up.